### PR TITLE
MAT-6269: Generating DRC Extensions in Bundle for Code entries in CQL

### PR DESCRIPTION
**Pull request by gregory-akins**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-6269](https://jira.cms.gov/browse/MAT-6269)
(Optional) Related Tickets:

### All Submissions
* [X] This PR has the JIRA linked.
* [ ] Required tests are included.
* [X] No extemporaneous files are included (i.e Complied files or testing results).
* [X] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/madiefhirservice/cql/LibraryCqlVisitor.java [hunk 1]
```diff
@@ -162,20 +176,20 @@ public String visitCodeDefinition(cqlParser.CodeDefinitionContext ctx) {
    */
   @Override
   public String visitRetrieve(cqlParser.RetrieveContext ctx) {
+    for (int i = 0; i < ctx.getChildCount(); i++) {
+      log.debug("Context {}", ctx.getChild(i).getText());
+    }
```
Comment (adongare) [inline src/main/java/gov/cms/madie/madiefhirservice/cql/LibraryCqlVisitor.java]: this is probably some code for debugging. can be removed. rest looks good!

**Review by adongare — APPROVED** [review]

**Review by jkotanchik-SB — APPROVED** [review]
